models/app.model.js

```diff
@@ -7,16 +7,34 @@ const fetchCategories = (request, response) => {
     })
 }
 
-const fetchReviews = () => {
-  
-    const reviewCommentCount = 
+const fetchReviews = (category, sort_by="created_at", order="desc") => {
+    const acceptedCategories = ["euro game", "social deduction", "dexterity", "children's games"];
```

Comment:
This is not maintainable! What if a category is added to our database? What if your site really takes off and _thousands_ of categories are added? You'd need to add them in here manually each time. Luckily, though, if all of our categories are stored in our database, we could check them by utilising our existing GET categories endpoint.